testsuite/run_forge.sh

```diff
@@ -175,9 +180,14 @@ get_dashboard_link() {
 
 print_output_files
 
+if [ -z "$FORGE_CLUSTER_NAME" ]; then
+    FORGE_CLUSTER_NAME=${FORGE_CLUSTERS[ $RANDOM % ${#FORGE_CLUSTERS[@]} ]}
+fi
```

Comment:
Why do we need this instead of just update kubeconfig?

Reply:
Since `kubectl` is stateful and remembers which context is currently configured, I'm a bit afraid of confusing ppl if running forge changes your `kubectl` context. In the ideal world, Forge runs *somewhere*, and devs don't really need to know where unless they're deep diving for debugging purposes.

Reply:
The forge cli itself though will continue to use whatever cluster you have locally so for that you would have to know where its running

This is something i can add to wrapper as well